demo/index.rtl.html

```diff
@@ -10,6 +10,7 @@
 		 */
 	</script>
 	<link rel="import" href="../d2l-table.html">
+	<link rel="import" href="../d2l-table-col-sort.html">
```

Comment:
You may just want to import `d2l-table-col-sort.html` from `d2l-table.html` so consumers only need to do the one import. Depends how much we want to "save" people who don't care about sorting.